test/unit/modules/node.js

```diff
@@ -313,40 +321,120 @@ describe('node', () => {
+							'loaded',
+							'networkHeight',
+							'syncing',
+						];
+						properties.forEach(property => {
+							expect(status).to.have.property(property);
+						});
+						expect(Object.keys(status).length).to.equal(properties.length);
+						done();
+					});
+				});
 
-				it(
-					'should call callback with result containing consensus = modules.peers.calculateConsensus result'
-				);
+				it('should call callback with result containing broadhash = modules.system.getBroadhash() result', done => {
+					node_module.shared.getStatus(null, (err, status) => {
+						expect(status.broadhash).to.equal(
+							library.modules.system.getBroadhash()
+						);
+						done();
+					});
+				});
 
-				it(
-					'should call callback with result containing height = modules.blocks.lastBlock.get result'
-				);
+				it('should call callback with result containing consensus = modules.peers.getLastConsensus() result', done => {
+					node_module.shared.getStatus(null, (err, status) => {
+						expect(status.consensus).to.equal(
+							library.modules.peers.getLastConsensus()
+						);
+						done();
+					});
+				});
 
-				it(
-					'should call callback with result containing syncing = modules.loader.syncing result'
-				);
+				it('should call callback with result containing height = modules.blocks.lastBlock.get().height result', done => {
+					node_module.shared.getStatus(null, (err, status) => {
+						expect(status.height).to.equal(
+							library.modules.blocks.lastBlock.get().height
+						);
+						done();
+					});
+				});
 
-				it('should call modules.loader.getNetwork');
+				it('should call callback with result containing syncing = modules.loader.syncing() result', done => {
+					node_module.shared.getStatus(null, (err, status) => {
+						expect(status.syncing).to.equal(library.modules.loader.syncing());
+						done();
+					});
+				});
 
-				describe('when modules.loader.getNetwork fails', () => {
-					it(
-						'should call callback with result containing networkHeight = null'
-					);
+				it('should call callback with result containing loaded = modules.loader.loaded() result', done => {
+					node_module.shared.getStatus(null, (err, status) => {
+						expect(status.loaded).to.equal(library.modules.loader.loaded());
+						done();
+					});
 				});
 
-				describe('when modules.loader.getNetwork succeeds and returns network', () => {
-					it(
-						'should call callback with result containing networkHeight = network.height'
-					);
+				describe('modules.loader.getNetwork', () => {
+					beforeEach(done => {
+						stubs.networkHeight = sinonSandbox
+							.stub()
+							.callsFake((filters, cb) => cb(null, 123));
+						const modules = library.rewiredModules.node.__get__('modules');
+						modules.peers.networkHeight = stubs.networkHeight;
+						done();
+					});
+
+					it('should call modules.peers.networkHeight and call callback with result containing networkHeight = network.height', done => {
```

Comment:
Looks like 2 separate tests for me - "call callback" and "call modules.peers.networkHeight"